### ISSUE TITLE
homeworks QS

### ISSUE THREAD

**Issue by sergiobaiao** [thecynic/pylutron#13]
hi there,

i just made pylutron working with my lutron homeworksQS setup. The only change i made was in __init__.py, where i changed GNET for QNET. The commands are all the same, both platforms rely on lutron QS Control Protocol. Can you include this on a future release so it can work with both systems?

**Comment by fl0zan0** [comment]
any news about that ?

**Comment by txwindsurfer** [comment]
Yes, I would really like to use this as well.  Can the prompt be a configuration variable?

**Comment by txwindsurfer** [comment]
I did the same and Hass now works great with HomeworksQS. You have to do a custom install however since this file isn't editable in hass.io

**Comment by fl0zan0** [comment]
Thank you very much

Best regards

**Comment by txwindsurfer** [comment]
Has #22 been closed? Doesn't appear that the code has been added to the base.

**Comment by txwindsurfer** [comment]
I am using Homeworks QS. Just FYI, the latest pylutron code works without hanging and without changing the prompt from GNET to QNET.

**Comment by teich** [comment]
Can confirm with a clean install of HassIO and HomeAssistant 0.101.3 Homeworks QS is working great with no s/G/Q needed. Thanks to however this got resolved!

**Comment by thecynic** [comment]
Bleh, that's actually a bug :-)

When I added timeouts to the code to handle connection drops, I inadvertently allowed it to just timeout on reading the prompt. So, it works because the repeater doesn't fail to connect us when user/pw is right :-)

I'll fix that to report errors correctly but will also add support for QS when I do.

**Comment by Unseen88** [comment]
@teich so you are saying if i do a clean hassio install above 0.101.3 and by using https://www.home-assistant.io/integrations/homeworks/ this as integration nothing else editing is required ?

**Comment by teich** [comment]
Yes, as of now QS works fine, due to the timeout bug above. I've been using hassio for months now without issue.

**Comment by rzulian** [comment]
@Unseen88 the integration you are referring to is only for series 4 & 8, if you have QS you need this one
https://www.home-assistant.io/integrations/lutron/

**Comment by cdheiser** [comment]
QS now works and is supported in the library
